assets/controllers/push_controller.js

```diff
@@ -106,7 +103,7 @@ export default class extends Controller {
                 return response.json()
             })
             .then(data => {
-
+                // Nothing?
```

Comment:
I think you can just leave this blank :D

Reply:
If you re not gonna use it, we can just remove this whole then.

Reply:
Removed the then. It can always be added, if we really want or need to.